exporters/prometheus/confg_test.go

```diff
@@ -25,9 +25,10 @@ func TestNewConfig(t *testing.T) {
 	registry := prometheus.NewRegistry()
 
 	testCases := []struct {
-		name           string
-		options        []Option
-		wantRegisterer prometheus.Registerer
+		name                  string
+		options               []Option
+		wantRegisterer        prometheus.Registerer
+		wantDisableTargetInfo bool
```

Comment:
Similar to [this](https://github.com/open-telemetry/opentelemetry-go/pull/3352/files#diff-f800849b0a317cd5ce38c44231bcbb488397953a23078433f0dba3b01c3190d2), can we have this be a `wantConfig` field?

Reply:
done.